psi4/driver/driver_cbs.py

```diff
@@ -1570,7 +1570,7 @@ def cbs(func, label, **kwargs):
         commands = '\n'
         commands += """\ncore.set_global_option('BASIS', '%s')\n""" % (mc['f_basis'])
         commands += """core.set_global_option('WRITER_FILE_LABEL', '%s')\n""" % \
-            (user_writer_file_label + ('' if user_writer_file_label == '' else '-') + mc['f_wfn'].lower() + '-' + mc['f_basis'].lower())
+            (user_writer_file_label + ('' if user_writer_file_label == '' else '-') + mc['f_wfn'].lower() + '-' + mc['f_basis'].lower().replace('*', 's'))
```

Comment:
Here's the [fn Jet mentioned](https://github.com/psi4/psi4/blob/master/psi4/driver/qcdb/basislist.py#L148-L161) in case you want to use, copy, or condense it. On the other hand, this is gone in the next-gen driver, and if it returns, the files will probably be sequestered by directory and not need these long filenames. So choice of fix is up to you.